responsibleai/responsibleai/_interfaces.py

```diff
@@ -38,7 +38,7 @@ class Dataset:
     object_detection_true_y: Optional[List]
     object_detection_predicted_y: Optional[List]
     imageDimensions: Optional[List[List[int]]]
-    object_detection_labels: Optional[List[str]]
+    object_detection_labels: Optional[List[Dict[str, str]]]
```

Comment:
This worries me a bit because we keep making breaking changes for object detection.  Now the old dashboards won't work at all anymore, if I understand correctly?
Might it be better at this point to just introduce a new "optional" variable to ensure that the old dashboards will still somehow render, or should we just go ahead and make this breaking change?

Reply:
Discussed offline, we wouldn't want the users to use the old dashboard as those Object Detection labels are incorrect.